src/drive/web/modules/services/components/SuggestionProvider.jsx

```diff
@@ -1,12 +1,11 @@
 /* global cozy */
 import React from 'react'
 import FuzzyPathSearch from '../FuzzyPathSearch'
-import { withClient, Q } from 'cozy-client'
+import { withClient } from 'cozy-client'
 
-import { DOCTYPE_FILES } from 'drive/lib/doctypes'
 import { TYPE_DIRECTORY, makeNormalizedFile } from './helpers'
 import { getIconUrl } from './iconContext'
-import { TRASH_DIR_ID } from 'drive/constants/config'
+import { buildSuggestionsQuery } from '../../queries'
```

Comment:
By doing this, you include the `queries` file inside the bundled intent. So you can have increase the bundle size without notifying it. 

Today, bundlemon doesn't take a look at `/targets/intents/` bundle size.

Reply:
> Today, bundlemon doesn't take a look at /targets/intents/ bundle size.
https://github.com/cozy/cozy-drive/pull/2664

Reply:
> By doing this, you include the queries file inside the bundled intent. So you can have increase the bundle size without notifying it.

Well done! 

<img width="576" alt="Capture d’écran 2022-08-05 à 16 15 50" src="https://user-images.githubusercontent.com/8363334/183095791-d97fab72-035d-4f0b-9f45-af3de0619b6b.png">
![Capture d’écran 2022-08-05 à 16 15 56](https://user-images.githubusercontent.com/8363334/183095802-40c7fba8-00f4-4251-b85a-8e97cf17a594.png)

Reply:
I remove the refacto(queries) commit then.

Reply:
I'm reading bad? We pass from 170KB to 169KB. I think we can let this commit, no?

Reply:
In local,  without the refacto:
```
 [PASS] drive/intents-drive.(hash).min.css: 36.11KB < 40KB
  [PASS] drive/intents/drive.(hash).js: 169.84KB < 171KB
  [PASS] drive/intents/drive.(hash).min.css: 36.11KB < 37KB
```
we have the same value.

So yes, we can let the commit